components/conversation/apiHelpers.js

```diff
@@ -108,13 +105,51 @@ const sendMessage = async (message, chatId) => {
   })
 };
 
+const startConversation = async (message, friendId) => {
+  const formData = new FormData();
+  formData.append('senderId', message.user._id);
+  formData.append('userId2', friendId);
+  formData.append('body', message.text);
+  return await fetch('http://3.133.100.147:2550/new-conversation', {
```

Comment:
Interesting you use both fetch and axios-- is there a preferred use case for one or the other?